lib/gvariant.nix

```diff
@@ -1,16 +1,17 @@
+/*
+  A partial and basic implementation of GVariant formatted strings.
+  See [GVariant Format Strings](https://docs.gtk.org/glib/gvariant-format-strings.html) for details.
+
+  :::{.warning}
+  This API is not considered fully stable and it might therefore
+  change in backwards incompatible ways without prior notice.
+  :::
+*/
+
```

Comment:
@linsui This now gets rendered in the Nixpkgs manual. Note that I changed the "Note" to a "Warning" here, does that sound good?

Reply:
Sounds good to me. :)